lambdas/service/app.py

```diff
@@ -85,7 +85,98 @@
 
 openapi_spec = {
     'info': {
-        'description': 'This should probably be really long',
+        'description': format_description('''
+
+            # Overview
+
+            Azul is a REST web service for querying metadata associated with
+            both experimental and analysis data stored in the [HCA Data Store
+            (DSS)](https://github.com/HumanCellAtlas/data-store). In order to
+            deliver response times that make it suitable for interactive use
+            cases, it is limited the set of metadata properties that it exposes
```

Comment:
I don't think this correct English. By passive voice I meant converting from "A limits B" to "B is limited".

Reply:
I forgot the preposition "in".

Reply:
But I decided to word it differently.